Sample.Api/App/AirportService.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Sample.Api.Model;
+
+namespace Sample.Api.App
+{
+	public class AirportService : HttpClient
+	{
+		public string ApiHost;

```

Comment:
Не совсем понятна идея, думаю лучше использовать только для чтения
```suggestion
		public string ApiHost {get;}

```